bedrock/firefox/templates/firefox/fights-for-you.de.html

```diff
@@ -106,6 +108,13 @@
   {% endcall %}
 </div>
 
+<div id="fffy-video-modal" class="mzp-c-card-video-wrapper">
```

Comment:
We should probably try and avoid using protocol classes outside of their parent components. I'd suggest renaming this to comething non-protocol specific?

Reply:
Hm, I was trying to use the protocol component. I'll have another look at the sample code.

Reply:
Oh I see, this is specific to the homepage and not part of the modal component. Updated!